blog/incident-report-week3/index.qmd

```diff
@@ -0,0 +1,30 @@
+---
+author: Mordred Boulais
+title: Incident Report Regarding First Professional Precis
+date: '2023-9-11'
+date-format: YYYY
+categories: [post, developer development, incident report, professional precis, SE1]
+description: <em>A discussion of what we struggled with and how to improve from our first incident.</em>
+---
+
+The overarching struggle with getting out our first blog post regarding `Software
+Engineering At Google` mostly centered on the team's lack of proper communication
+and mutual understanding - of Github and of the task at hand. This was amplified
+by a lack of secure and consistent access to WiFi, and of a universal
+understanding of the terms used to describe the task.
+
+In order to tackle these issues, the team laid out a new methodology for the
+completion of tasks, involving the assignment of smaller committees to each
+task. This is intended to facillate closer and more accurate communication among
+teammates and to ensure clarity of assigned work. Part of obtaining this clarity
+is to assign each committee a 'Github Expert', someone confident enough in their
+understanding of Github flow and functions to help guide other committee members.
```

Comment:
Please link to the web site that explains GitHub Flow. Please note that there is not a mandate that the team members must follow GitHub Flow. However, it is a nice way to organize collaboration and one that I have used on several prior projects.